<ui version="4.0" >
 <class>widgetMedia</class>
 <widget class="QWidget" name="widgetMedia" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Media Type</string>
  </property>
  <widget class="QRadioButton" name="radioCD" >
   <property name="geometry" >
    <rect>
     <x>72</x>
     <y>100</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>CD</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioSL" >
   <property name="geometry" >
    <rect>
     <x>72</x>
     <y>160</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>DVD SL (4 gb)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioDL" >
   <property name="geometry" >
    <rect>
     <x>72</x>
     <y>220</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>DVD DL (8 gb)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioExternal" >
   <property name="geometry" >
    <rect>
     <x>72</x>
     <y>280</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>External Disk</string>
   </property>
  </widget>
  <widget class="QLabel" name="widgetDesc" >
   <property name="geometry" >
    <rect>
     <x>52</x>
     <y>20</y>
     <width>420</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p align="center" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Select target media type&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>310</y>
     <width>270</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QSlider" name="sliderSize" >
      <property name="maximum" >
       <number>500</number>
      </property>
      <property name="pageStep" >
       <number>10</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinSize" >
      <property name="readOnly" >
       <bool>false</bool>
      </property>
      <property name="maximum" >
       <number>500</number>
      </property>
      <property name="singleStep" >
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderSize</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>179</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel" >
     <x>298</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>300</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
